modules/task/execution_test.go

```diff
@@ -135,6 +142,7 @@ func TestEnqueueSimpleFn(t *testing.T) {
 	err = <-_errorCh
 	require.Error(t, err)
 	assert.Contains(t, err.Error(), "Simple error")
+	verifyMetrics(t, _testScope)
```

Comment:
pulling this out into a function is not totally helpful, since it's a one-and-done function that can't be easily reused